docs/cypress/support/commands.ts

```diff
@@ -34,4 +34,38 @@
 //       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
 //     }
 //   }
-// }
\ No newline at end of file
+// }
+Cypress.Commands.add('tableOfContent', () => {
+    cy.wait(3000);
+    cy.get('[data-test=Docs-Toc--Link]').each((page) => {
+        cy.get('.Heading.Heading--default.mr-4').contains(`${page.text()}`);
+    });
+});
+
+Cypress.Commands.add('imagesCheck', () => {
+    cy.viewport('macbook-15');
+    cy.wait(3000)
+
+    cy.get('img').each(($img) => {
+        cy.wrap($img).scrollIntoView().should('be.visible').should('have.attr', 'alt');
+    });
+});
+Cypress.Commands.add('tileClick', () => {
+    cy.wait(1000);
+    cy.get('.Tile').click({
+        multiple: true,
+    });
+});
+
+Cypress.Commands.add('linkCheck', () => {
```

Comment:
Please update the function name instead of using `linkCheck`, `tileCheck`, `imagesCheck`